px-k8s-sql-ha/intro.md

```diff
@@ -0,0 +1,8 @@
+[Portworx](https://portworx.com/) is a cloud-native storage fabric designed to enable enterprise containers such as databses in production. Portworx integrates seamlessly with container orchestrators such as Kubernetes, Marathon, Nomad and Swarm. Portworx allows containerized stateful applications such as Microsoft SQL Server to programmatically consume block, file and object storage services directly through the scheduler.
+
+In this tutorial, you will learn:
+* How to deploy SQL Server on Kubernetes with multi-node High-Availability
+* How to inspect details of a Portworx Volume
```

Comment:
The scenario doesn't inspect the portworx volume. The volume inspect part is already covered in https://www.katacoda.com/portworx/scenarios/px-k8s-vol-basic